votes/0793-Tristan1900-and-RidRisR-as-tidb-reviewers.md

```diff
@@ -0,0 +1,19 @@
+# A Vote for My Proposal
```

Comment:
Please rename this file to `0801-Tristan1900-and-RidRisR-as-tidb-reviewers.md`

Reply:
done